src/shared/components/oncoprint/controls/OncoprintControls.tsx

```diff
@@ -609,11 +611,11 @@ export default class OncoprintControls extends React.Component<
         const showGenesTextArea =
             showItemSelectionElements &&
             this.props.state.selectedHeatmapProfileAlterationType !==
-                AlterationTypeConstants.TREATMENT_RESPONSE;
+                AlterationTypeConstants.GENERIC_ASSAY;
         const showTreatmentsTextArea =
             showItemSelectionElements &&
             this.props.state.selectedHeatmapProfileAlterationType ===
```

Comment:
So, here it should state something to the effect of:
```
this.props.state.selectedHeatmapProfileAlterationType === AlterationTypeConstants.GENERIC_ASSAY
&& this.props.state.genericAssayType === GenericAssayTypeConstants.TREATMENT_RESPONSE
```
or
```
this.props.state.selectedHeatmapProfileType === GenericAssayTypeConstants.TREATMENT_RESPONSE
```
I know this is not possible in the current implementation of the interface, but we should from now on handle generic assays by their real type rather than GENERIC_ASSAY type.